csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -778,6 +785,11 @@ static Settings()
             /// </summary>
             public TypeRegistry TypeRegistry { get; }
 
+            /// <summary>
+            /// Whether to format enums as ints. Defaults to false.
+            /// </summary>
+            public bool FormatEnumsAsIntegers { get; }
+
```

Comment:
We can remove the line below this now - I hadn't spotted it earlier. At least it means I'm consistent :)